spec/raw_coverage_spec.rb

```diff
@@ -7,27 +7,36 @@
     describe "with two faked coverage resultsets" do
       before do
         @resultset1 = {
-          source_fixture("sample.rb") => [nil, 1, 1, 1, nil, nil, 1, 1, nil, nil],
-          source_fixture("app/models/user.rb") => [nil, 1, 1, 1, nil, nil, 1, 0, nil, nil],
-          source_fixture("app/controllers/sample_controller.rb") => [nil, 1, 1, 1, nil, nil, 1, 0, nil, nil],
-          source_fixture("resultset1.rb") => [1, 1, 1, 1],
-          source_fixture("parallel_tests.rb") => [nil, 0, nil, 0],
-          source_fixture("conditionally_loaded_1.rb") => [nil, 0, 1],  # loaded only in the first resultset
-          source_fixture("three.rb") => [nil, 1, 1],
+          source_fixture("sample.rb") => {
+            :lines => [nil, 1, 1, 1, nil, nil, 1, 1, nil, nil],
+            :branches => {[:if, 3, 8, 6, 8, 36] => {[:then, 4, 8, 6, 8, 12] => 47, [:else, 5, 8, 6, 8, 36] => 24}},
```

Comment:
Was there supposed to be a corresponding fixture change in this PR? Or is this intended to be fake branch data?

Reply:
Will be covered in the release [issue](https://github.com/colszowka/simplecov/issues/781)